src/testdir/test_increment.vim

```diff
@@ -796,4 +796,52 @@ func Test_increment_special_char()
   call assert_beeps("normal \<C-X>")
 endfunc
 
+" Try incrementing/decrementing a number when nrformats contains unsigned
+func Test_increment_unsigned()
+  set nrformats+=unsigned
+
+  " 18446744073709551615 == pow(2, 64) - 1
+  if 64 == v:numbersize
```

Comment:
Now `v:numbersize` is always 64.

Reply:
Is not this statement unnecessary?

Reply:
When I added the num64 feature, the size of Number was depended on the platform. But 8.2.0271 enabled the num64 feature for all platforms. And after that, `v:numbersize` was added in 8.2.0296.
I don't think that the "if" statement is needed. (Or are you concerned about that v:numbersize is more than 64?)